game.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+# Game
+class Game
+  def initialize
+    puts ''
+  end
+
+  def start
+    show_welcome
+    show_message
+  end
+
+  def show_welcome
+    puts '#' * 29
+    print '#' * 10, ' Welcome ', '#' * 10, "\n"
+    puts '#' * 29
+  end
+
+  def show_message
```

Comment:
whats mean this text?

Reply:
Its just some message example that i found.